.github/workflows/trunk.yml

```diff
@@ -271,6 +271,62 @@ jobs:
           exit 1
         fi
 
+  test-arm-cortex-m-zephyr-size-test:
```

Comment:
can we deduplicate and merge this job with `test-arm-cortex-m-size-test`, see `matrix` above